setup.cfg

```diff
@@ -1,3 +1,50 @@
+[metadata]
+name = httpx
+version = attr: src.get_version
+url = https://github.com/encode/httpx
+license = BSD
+description = The next generation HTTP client.
+long_description = file: README.md, file: CHANGELOG.md
+long_description_content_type = text/markdown
+author = Tom Christie
+author_email = tom@tomchristie.com
+classifiers =
+    Development Status :: 3 - Alpha
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Topic :: Internet :: WWW/HTTP
+    Framework :: AsyncIO
+    Framework :: Trio
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+
+[options]
+python_requires = >=3.6
+packages = find:
+include_package_data = True
```

Comment:
From my reading of the spec you don't need this if you specify `[options.package_data]`